packages/plugins/ENS/src/SNSAdaptor/SearchResultInspector.tsx

```diff
@@ -0,0 +1,65 @@
+import { useContext } from 'react'
+import { ChainId } from '@masknet/web3-shared-evm'
+import { SourceType } from '@masknet/web3-shared-base'
+import { Box, Typography } from '@mui/material'
+import useStyles from './useStyles'
+import { useI18N } from '../locales'
+import { Icons } from '@masknet/icons'
+import { EmptyContent } from './EmptyContent'
+import { LoadingContent } from './LoadingContent'
+import { LoadFailedContent } from './LoadFailedContent'
+import { TopAndLastOffers } from './TopAndLastOffers'
+import { DataSourceSwitcherWrapper } from './DataSourceSwitcherWrapper'
+import { ENSProvider, ENSContext, SearchResultInspectorProps } from './context'
+import { CollectibleState } from './hooks/useCollectibleState'
+
+export function SearchResultInspectorContent() {
+    const t = useI18N()
+    const { classes } = useStyles()
+    const { isLoading, isNoResult, isError, reversedAddress, retry, nextIdTwitterBindingName, domain, tokenId } =
+        useContext(ENSContext)
+
+    if (isNoResult || !reversedAddress || !tokenId) return <EmptyContent />
+
+    if (isLoading) return <LoadingContent />
+
+    if (isError) return <LoadFailedContent isLoading={isLoading} retry={retry} />
+    return (
+        <CollectibleState.Provider
+            initialState={{
+                chainId: ChainId.Mainnet,
+                tokenId,
+                contractAddress: reversedAddress,
+                provider: SourceType.OpenSea,
+            }}>
+            <Box className={classes.root}>
+                <div className={classes.coverCard}>
+                    <Icons.ENSCover className={classes.coverIcon} />
```

Comment:
`ENSCover.png` should not be treated as an icon.

Reply:
done